fury/interactor.py

```diff
@@ -142,25 +144,57 @@ def _process_event(self, obj, evt):
             self.on_mouse_wheel_forward(obj, evt)
         elif evt == "MouseWheelBackwardEvent":
             self.on_mouse_wheel_backward(obj, evt)
+        # elif evt == "LeftButtonDoubleClickEvent":
+        #     self.on_left_button_double_click(obj, evt)
 
```

Comment:
To differentiate Single clicks and double clicks wouldn't i need custom events like these?

Reply:
Or we could pass two methods, one for Single click and the other for double clicks...

Reply:
I believe we can pass 2 methods, to test, and then, we will have to create a new event.
doubleclick should cancel singleclick action/propagation when it is detected